uninstall.php

```diff
@@ -186,6 +186,7 @@ public function uninstall() {
 
 		// Delete transients.
 		delete_transient( 'pll_languages_list' );
+		delete_option( '_transient_pll_languages_list' ); // Force deletion of the transient from the options table in case external object cache is used.
```

Comment:
```suggestion
		delete_option( '_transient_pll_languages_list' ); // Force deletion of the transient from the options table in case external object cache is used.
```
We don't need to delete the external cache in case of clean uninstall. Do we?

Reply:
Done in 3e66676a.